### PR TITLE
[BD-6664] Update bpk-mixins and bpk-stylesheets for input placeholder color

### PR DESCRIPTION
**Pull request by Supremeyh**
##  Context
We have recived some [feedback](https://skyscanner.slack.com/archives/C03V4AJ8VQB/p1675840752294329) that the default color of placeholder in BpkInput looks pretty light-colored,  which might make the content unclearly. After [discuss](https://skyscanner.slack.com/archives/CUQ587E5D/p1675857743401299), decided make it darker, according the [figma](https://www.figma.com/file/irZ3YBx8vOm16ICkAr7mB3/Backpack?node-id=6353%3A6397&t=RMotY3araZaFAYDx-0), we update the placeholder color. In the [backpack foundations pr](https://github.com/Skyscanner/backpack-foundations/pull/190) we have updated the color, and this step forward to update `bpk-mixins` and `bpk-stylesheets` version which being used in backpack components.

## Change Description
 - Update `bpk-mixins` and `bpk-stylesheets` for input placeholder color.

### Screenshots

| Before                        | After                         |
| ----------------------------- | ----------------------------- |
|  |  |

Remember to include the following changes:

- [x] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack/blob/main/CHANGELOG_FORMAT.md))
- [ ] `README.md`
- [ ] Tests
- [ ] Storybook examples created/updated

### CODE REVIEW

**Comment by anambl** [comment]
Might be worth updating the screenshots for input and text area component! Not sure what the difference is in the card component and why did Storybook picked it up? 😨

**Comment by Supremeyh** [comment]
> Might be worth updating the screenshots for input and text area component! Not sure what the difference is in the card component and why did Storybook picked it up? 😨

Yeah, the screenshots for input and text area component should have been updated. Opposite for the card component. However, tried a few times on locally and drone, it still only the same result, otherwise cannot pass these test cases. I have checked the [diff](https://github.com/Skyscanner/backpack-foundations/compare/bpk-stylesheets%409.0.1...bpk-stylesheets%409.0.2), no other visual changes included.  Maybe the change for placeholder color is big difference so snapshot cannot captured.  How about update placeholder content in text and textarea to `Search country, city or airport` so the it be updated compulsively.

**Comment by anambl** [comment]
> > Might be worth updating the screenshots for input and text area component! Not sure what the difference is in the card component and why did Storybook picked it up? 😨
> 
> Yeah, the screenshots for input and text area component should have been updated. Opposite for the card component. However, tried a few times on locally and drone, it still only the same result, otherwise cannot pass these test cases. I have checked the [diff](https://github.com/Skyscanner/backpack-foundations/compare/bpk-stylesheets%409.0.1...bpk-stylesheets%409.0.2), no other visual changes included. Maybe the change for placeholder color is big difference so snapshot cannot captured. How about update placeholder content in text and textarea [examples](https://github.com/Skyscanner/backpack/blob/main/examples/bpk-component-input/examples.js#L114) to such like `Search city, country or airport` so the it be updated compulsively.

Sure, feel free to change the placeholder text!

File: examples/bpk-component-input/examples.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ const TextExample = () => (
       name="text_value"
       value="Edinburgh"
       onChange={action('input changed')}
-      placeholder="Enter a country, city or airport"
+      placeholder="Enter a country, city or airport..."
```
Comment (olliecurtis) [inline examples/bpk-component-input/examples.js:101]: Just checking what the purpose of adding the `...` at the end of this is for?

File: UNRELEASED.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-`@skyscanner/backpack-web`
+`bpk-mixins`: `39.0.1=> 39.1.0`
+`bpk-stylesheets`: `9.0.1 => 9.0.2`
```
Comment (olliecurtis) [inline UNRELEASED.md:2]: We don't need to add these things here as its the internals to the below that will be the change :)

File: UNRELEASED.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 
 **Changed:**
-  - `BpkDividedCard`
-    - Add prop `isElevated` to switch on/off the card shadow so that it wouldn't overlap with the shadow of the card wrapper.
```
Comment (olliecurtis) [inline UNRELEASED.md:5]: This change is still needed as it hasn't been released yet so will require being added back

**Review by olliecurtis — CHANGES_REQUESTED** [review]

File: examples/bpk-component-input/examples.js (hunk 1 above)
Comment (Supremeyh) [reply examples/bpk-component-input/examples.js]: It due to the snapshots cannot be updated at the beginning, so changed the placeholder content here. It seems work now and revise these change.

File: UNRELEASED.md (hunk 2 above)
Comment (Supremeyh) [reply UNRELEASED.md]: Thanks, it saw some similar changes in [commit](https://github.com/Skyscanner/backpack/pull/2633/files#diff-2963e3ff282a7ad8e0a661410ad57dddfbdae0ffdcc6031a8d65d1d468a56d69R1), and have removed these strings now.

File: UNRELEASED.md (hunk 3 above)
Comment (Supremeyh) [reply UNRELEASED.md]: Sure thing, will add them back.

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**